test/units/TEST-30-ONCLOCKCHANGE.sh

```diff
@@ -20,7 +20,14 @@ timedatectl set-timezone Europe/Kyiv
 
 while test ! -f /tmp/timezone-changed ; do sleep .5 ; done
 
-timedatectl set-time 2018-1-1
+# Get current time and add a future offset (e.g., 1 year and 1 month ahead)
```

Comment:
Please also update one more `set-time` call in the above.

Reply:
done, I updated that as well, could you please verify.